x-pack/plugins/security_solution/public/resolver/store/ui/selectors.ts

```diff
@@ -1,30 +1,50 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { createSelector } from 'reselect';
-import { ResolverUIState } from '../../types';
-
-/**
- * id of the "current" tree node (fake-focused)
- */
-export const ariaActiveDescendant = createSelector(
-  (uiState: ResolverUIState) => uiState,
-  /* eslint-disable no-shadow */
-  ({ ariaActiveDescendant }) => {
-    return ariaActiveDescendant;
-  }
-);
-
-/**
- * id of the currently "selected" tree node
- */
-export const selectedNode = createSelector(

```

Comment:
![image](https://user-images.githubusercontent.com/35559/91485308-a890c180-e878-11ea-855e-9f37c77688af.png)
I'm guessing this is another line-endings thing.